slobrok/src/vespa/slobrok/server/sbenv.cpp

```diff
@@ -116,15 +116,23 @@ SBEnv::SBEnv(const ConfigShim &shim)
       _rpcsrvmap()
 {
     srandom(time(nullptr) ^ getpid());
-    _rpcsrvmap.proxy().registerListener(_globalVisibleHistory);
-    _rpcsrvmap.proxy().registerListener(_localRpcMonitorMap);
+    _localMonitorSubscription = MapSubscription::subscribe(_rpcsrvmap.proxy(), _globalVisibleHistory);
+    _consensusSubscription = MapSubscription::subscribe(_rpcsrvmap.proxy(), _localRpcMonitorMap);
+    _globalHistorySubscription = MapSubscription::subscribe(_localRpcMonitorMap.dispatcher(), _consensusMap);
```

Comment:
have the _globalHistorySubscription and _localMonitorSubscription subscriptions been swapped? (got a bit confused by the names...)

Reply:
> have the _globalHistorySubscription and _localMonitorSubscription subscriptions been swapped?

not sure how I managed to swap these, should be fixed now.